#!/usr/bin/python3
def raise_exception_msg(message=""):
    """NameError exception with msg."""
    raise NameError(message)
